src/app/utilities/api-clients/datasets.js

```diff
@@ -24,9 +24,33 @@ export default class datasets {
              });
     }
 
+    static getVersion(datasetID, edition, version) {
+        return http.get(`/dataset/datasets/${datasetID}/editions/${edition}/versions/${version}`)
+             .then(response => {
+                 return response;
+             });
+    }
+
+    static getVersionDimensions(datasetID, edition, version) {
+        return http.get(`/dataset/datasets/${datasetID}/editions/${edition}/versions/${version}/dimensions`)
+             .then(response => {
+                 return response;
+             });
+    }
+
+    static updateVersion(datasetID, edition, version) {
+        const body = {
+        }
+        return http.put(`/datasets/${datasetID}/editions/${edition}/versions/${version}`, body, true)
```

Comment:
I think this is missing `/dataset` at the beginning of it. I'm currently getting an error when I save a version and it looks as though it's because it's not speaking to the dataset API.

Reply:
Yep, added that in.